frontend/src/component/project/Project/EditProject/EditProject.tsx

```diff
@@ -68,6 +80,12 @@ const EditProject = () => {
                 setToastApiError(formatUnknownError(error));
             }
         }
+
+        if (Boolean(projectScopedStickiness) && projectStickiness !== '') {
```

Comment:
I'd say it should be behind a stickiness setting hook without leaking out the details to the client code